KinSDK/KinSDK/BulkPayments/TransactionInterceptor.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  TransactionInterceptor.swift
+//  KinSDK
+//
+//  Created by Corey Werner on 13/08/2019.
+//  Copyright © 2019 Kin Foundation. All rights reserved.
+//
+
+import Foundation
+
+public protocol TransactionInterceptor {
+    func interceptTransactionSending(process: TransactionProcess) throws -> TransactionId
```

Comment:
add the comment from the tech design

Reply:
will do this apart of the dedicated ticket